Yoast/wordpress-seo
# Embed rssfooter integration

## Summary


This PR can be summarized in the following changelog entry:

* N/A - Just moved code to its own integration

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Do a `composer install`
* Go to admin -> yoast -> search appearance -> rss and set values for rss before and rss after.
* Go to the `<the domain you are testing on>`.test/feed and see it still being rendered.

## UI changes

## Documentation

## Quality assurance


Fixes #
